README.md

```diff
@@ -2,7 +2,7 @@
 
 Scholar@UC is a digital repository that enables the University of Cincinnati community to share its research and scholarly work with a worldwide audience. Faculty and staff can use Scholar@UC to collect their work in one location and create a durable and citeable record of their papers, presentations, publications, datasets, or other scholarly creations.
 
-Scholar@UC uses Project Hydra's [Hyrax gem](https://github.com/projecthydra-labs/hyrax). The source for UC's Hyrax-based app can be found at https://github.com/uclibs/scholar_uc
+Scholar@UC uses Project Samvera's [Hyrax gem](https://github.com/projecthydra-labs/hyrax). The source for UC's Hyrax-based app can be found at https://github.com/uclibs/scholar_uc
```

Comment:
@lawhorkl When the name changed from "Project Hydra" they dropped the "Project" part.  So replace "Project Hydra" with just "Samvera" wherever the change is needed.